sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestAssistantMessage.java

```diff
@@ -207,11 +223,12 @@ public static ChatRequestAssistantMessage fromJson(JsonReader jsonReader) throws
     /**
      * Creates an instance of ChatRequestAssistantMessage class.
      *
-     * @param content the content value to set.
+     * @param content the Binary content value to set.
      */
-    @Generated
     public ChatRequestAssistantMessage(BinaryData content) {
```

Comment:
We may want to make this constructor private, seeing that you have already added a couple of convenience ctxs with types that are more user friendly.